Healthy/Classes/ReusableViews/FormTextField/FormTextField.swift

```diff
@@ -0,0 +1,78 @@
+
+import UIKit
+
+class FormTextField: UIView {
+    
+    // MARK: - Outlets
+    //
+    
+    @IBOutlet weak private var contentView: UIView!
+    @IBOutlet weak private var titleLabel: UILabel!
+    @IBOutlet weak private var inputTextField: UITextField!
+    @IBOutlet weak private var errorLabel: UILabel!
+    
+    // MARK: - Properties
+    //
+    var text: ((String)->())?
+    
+    // MARK: - init
+    //
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        initView()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        initView()
+    }
+    
+    private func initView() {
+        Bundle.main.loadNibNamed(FormTextFieldResources.nibName, owner: self, options: nil)
+        addSubview(contentView)
+        contentView.frame = self.bounds
+        contentView.autoresizingMask = [.flexibleHeight, .flexibleWidth]
+        setupDesign()
+        bindToTextFieldsChanges()
+    }
+}
+
+// MARK: - configure
+//
+
+extension FormTextField {
+    func configure(title: String, placeholder: String?, type: UIKeyboardType) {
```

Comment:
What if we need to configure the keyboard type only? placeholder? Let's create a computed property for each property instead.